include/NAS2D/Resources/Sprite.h

```diff
@@ -111,35 +99,15 @@ class Sprite
 
 private:
 	/**
-	 * \class	spriteFrame
+	 * \struct	SpriteFrame
 	 * \brief	Contains
 	 */
-	class SpriteFrame
+	struct SpriteFrame
 	{
-	public:
-		SpriteFrame(const std::string& sheetId, int x, int y, int w, int h, int aX, int aY, int d);
-		SpriteFrame(const SpriteFrame &spriteframe) = default;
-		SpriteFrame& operator=(const SpriteFrame& rhs) = default;
-		~SpriteFrame() = default;
-
-		const std::string& sheetId() const { return mSheetId; }
-
-		int anchorX() const { return mAnchorX; }
-		int anchorY() const { return mAnchorY; }
-
-		int width() const { return mRect.width(); }
-		int height() const { return mRect.height(); }
-		int x() const { return mRect.x(); }
-		int y() const { return mRect.y(); }
-
-		int frameDelay() const { return mFrameDelay; }
-
-	private:
-		std::string mSheetId;
-		int mFrameDelay;
-		int mAnchorX;
-		int mAnchorY;
-		Rectangle_2d mRect;
+		std::string sheetId;
+		Rectangle<int> bounds;
+		Vector<int> anchor;
```

Comment:
A `Vector` was chosen because the value is used as a displacement. That was why in the constructor I had it named as `anchorOffset`. Though the constructor got removed.

In regards to `bounds`, the anchor is relative to the `startPoint`. We need to either rename this field, or change what it represents if we want to make it a `Point`.